pmd-cli/src/main/java/net/sourceforge/pmd/cli/commands/internal/PmdCommand.java

```diff
@@ -140,9 +143,20 @@ public void setBenchmark(final boolean benchmark) {
         this.benchmark = benchmark;
     }
 
-    @Option(names = "--short-names", description = "Prints shortened filenames in the report.")
-    public void setShortnames(final boolean shortnames) {
-        this.shortnames = shortnames;
+    @Option(names = { "--relativize-paths-with", "-z"}, description = "Path relative to which directories are rendered in the report. "
```

Comment:
this should be a `@Parameters` as it takes an argument

Reply:
not sure, but maybe `arity`? Currently, if you want to provide two paths, you need to use `-z path1 -z path2`. I think, that's different compared to PMD6 (but again, PMD7 is a major version change, so you can't expect it to work as a drop-in replacement).

With `arity="1..*"` one could use `-z path1 path2`. I'm unsure however, what to use as split... probably the same like for `--dir`, which would be `split = ","`...

I think, I'll change it to use arity+split, that makes the cli more consistent.

Reply:
PicoCli support a parameter being both multiple arity and repeatable… however, multiple arity parameters mess up with autocomplete, as it can't tell if what you are writing next is another option or element, so I'd avoid the variable arity if we can

Reply:
if using `@Parameter` here, wouldn't this conflict with the other parameter?

https://github.com/pmd/pmd/blob/6ff8bda8bc76b8f082d64cad50b86192b0eeb250/pmd-cli/src/main/java/net/sourceforge/pmd/cli/commands/internal/AbstractAnalysisPmdSubcommand.java#L50-L52

Reply:
yeah, you are right, sorry about that.